packages/datatrak-web-server/src/utils/formatTaskChanges.ts

```diff
@@ -12,24 +14,37 @@ type Output = Partial<Task> & {
   comment?: string;
 };
 
-export const formatTaskChanges = (task: Input) => {
-  const { due_date: dueDate, repeat_schedule: repeatSchedule, ...restOfTask } = task;
+const convertDateToEndOfDay = (date: Date | number) => {
```

Comment:
Any reason to not just use date-fns endOfDay here?

Reply:
No real reason except that it was simple enough to do with vanilla js